### DISCUSSION TITLE
[Idea] ComboBox / Dropdown / Spinner

### DISCUSSION THREAD

**Discussion by jor1196** [syncfusion/maui-toolkit#53] [Ideas]
I think this element was always requested but never incorporated, syncfusion is one of the most complete and serious toolkit that could fulfill the desire of many. It could have features like:

- ItemSource
- SeletedItem
- ItemDisplay
- SelectedCommand
- Dropdown MaxHeight
- Dropdown MinHeight
- or a higher characteristic, which allows us to define the dataTemplate to display the itemSource

**Comment by PaulAndersonS** [comment]
Hi @jor1196 ,

Are you expecting this ComboBox control from Syncfusion in toolkit? https://help.syncfusion.com/maui/combobox/overview

**Reply by jor1196** [reply]
Yep
